main.py

```diff
@@ -77,35 +77,14 @@ def process_keyboard_events(actions_while_key_pressed, actions_on_key_down):
             action.execute_action()
 
 
-def process_keyboard_events(actions_while_key_pressed, actions_on_key_down):
-    for event in pygame.event.get():
-        if event.type == pygame.locals.QUIT:
-            pygame.quit()
-            sys.exit()
-        elif event.type == pygame.locals.KEYDOWN:
-            for action in actions_on_key_down:
-                if event.key == action.trigger_key.key_identifier:
-                    action.execute_action()
-
-    # check all key states and automatically perform all callbacks
-    key_press_state = pygame.key.get_pressed()
-    for action in actions_while_key_pressed:
-        trigger_action_given: bool = (
-            action.trigger_key.pressed
-            == key_press_state[action.trigger_key.key_identifier]
-        )
-        if trigger_action_given:
-            action.execute_action()
-
-
 def main():
     pygame.init()
     game_window = GameWindow("Landing Game")
     game_timing = GameTiming()
     rocket_pos = Vec2d(
         CommonConstants.WINDOW_WIDTH / 2, CommonConstants.WINDOW_HEIGHT / 2
     )
-    rocket_mass = 1e5
+    rocket_mass = CommonConstants.ROCKET_MASS
```

Comment:
Is the mass of the Rocket a Constant?
I could imagine scenarios where you have to land with different rockets (having different masses) or losing mass while burning fuel in the future.

Reply:
This is physically correct and has its own differential equations, see e.g.
https://www.leifiphysik.de/mechanik/impulserhaltung-und-stoesse/grundwissen/raketenphysik!

For now, this is out of scope. Additionally, fuel mass would be added to the _constant_ mass of the rocket's body.
So I see nothing wrong storing that in a variable and maybe add current fuel-mass in the future.

Reply:
Leifi!
Ok, you are right